var/spack/repos/builtin/packages/root/package.py

```diff
@@ -253,7 +253,7 @@ class Root(CMakePackage):
     depends_on('veccore',   when='+veccore')
     depends_on('vdt',       when='+vdt')
     depends_on('libxml2+python',   when='+xml+python')
```

Comment:
I think you want to remove that line, too
```suggestion
```

Reply:
I *thought* I'd checked that ROOT's Python bindings required libxml2's Python bindings, but my neurons are failing, so I will double-check.

Reply:
I have confirmed with ROOT devs that ROOT's Python bindings do not in turn require libxml2's Python bindings. I will amend the PR.

Reply:
Thanks a lot for checking with upstream!